File: namsor/ruby/src/CastegroupIndianFullExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["api_key"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::IndianApi.new.castegroup_indian_full(
        "IN-UP", # sub_division_iso31662
        "Akash Sharma", # personal_name_full
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling IndianApi#castegroup_indian_full: #{e}"
end

File: namsor/php/src/CastegroupIndianFullExample.php
<?php

namespace OSEG\NamsorExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Namsor;

$config = Namsor\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("X-API-KEY", "YOUR_API_KEY");

try {
    $response = (new Namsor\Client\Api\IndianApi(config: $config))->castegroupIndianFull(
        sub_division_iso31662: "IN-UP",
        personal_name_full: "Akash Sharma",
    );

    print_r($response);
} catch (Namsor\Client\ApiException $e) {
    echo "Exception when calling IndianApi#castegroupIndianFull: {$e->getMessage()}";
}
